Application/EdFi.Ods.SandboxAdmin/Initialization/InitializationSettingsFactory.cs

```diff
@@ -0,0 +1,54 @@
+﻿// SPDX-License-Identifier: Apache-2.0
+// Licensed to the Ed-Fi Alliance under one or more agreements.
+// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// See the LICENSE and NOTICES files in the project root for more information.
+
+using System.Configuration;
```

Comment:
This is a clear indication that this class is an issue. in net core we are not using xml

Reply:
removed